test/groovy/util/JenkinsLoggingRule.groovy

```diff
@@ -38,12 +38,12 @@ class JenkinsLoggingRule implements TestRule {
             @Override
             void evaluate() throws Throwable {
 
-                testInstance.helper.registerAllowedMethod("echo", [String.class], {
+                testInstance.helper.registerAllowedMethod("echo", [String], {
```

Comment:
[minor] Having the `.class` is not explicitly wrong. This puts noise on the PR.

Reply:
Sorry but as I clean my campground I addressed some code style issues and did very small cleanups in that context.